### PR TITLE
move opae samples to directory

### PR DESCRIPTION
**Pull request by anandaravuri**

### CODE REVIEW

File: samples/hello_events/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    LIBS
+        safestr
+        opae-c
+        ${libjson-c_LIBRARIES}
+        ${libuuid_LIBRARIES}
+    COMPONENT samplebin
+)
+
+set(OPAE_SAMPLES src/opae/samples/hello_events)
+set(SAMPLES_SRC hello_events.c)
```
Comment (tswhison) [inline samples/hello_events/CMakeLists.txt]: Don't really need these extra variables, do we?

File: samples/object_api/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    LIBS
+        safestr
+        opae-c
+        ${libjson-c_LIBRARIES}
+        ${libuuid_LIBRARIES}
+    COMPONENT samplebin
+)
+
+set(OPAE_SAMPLES src/opae/samples/object_api)
+set(SAMPLES_SRC object_api.c)
```
Comment (tswhison) [inline samples/object_api/CMakeLists.txt]: Don't really need these extra variables, do we?

File: samples/hello_fpga/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    LIBS
+        safestr
+        opae-c
+        ${libjson-c_LIBRARIES}
+        ${libuuid_LIBRARIES}
+    COMPONENT samplebin
+)
+
+set(OPAE_SAMPLES src/opae/samples/hello_fpga)
+set(SAMPLES_SRC hello_fpga.c)
```
Comment (tswhison) [inline samples/hello_fpga/CMakeLists.txt]: Don't really need these extra variables, do we?

File: samples/hello_events/CMakeLists.txt (hunk 1 above)
Comment (anandaravuri) [reply samples/hello_events/CMakeLists.txt]: fixed,  removed variable

File: samples/object_api/CMakeLists.txt (hunk 2 above)
Comment (anandaravuri) [reply samples/object_api/CMakeLists.txt]: fixed,  removed variable

File: samples/hello_fpga/CMakeLists.txt (hunk 3 above)
Comment (anandaravuri) [reply samples/hello_fpga/CMakeLists.txt]: fixed,  removed variable

**Pull request merged**